staging/kos/cmd/controller-manager/main.go

```diff
@@ -55,21 +55,21 @@ func main() {
 		os.Exit(2)
 	}
 
-	kosInformersClientset, err := kosclientset.NewForConfig(kosClientCfg)
+	kosInformersClientset, err := kosclientset.NewForConfig(addUserAgent(kosClientCfg, "kos-controller-manager"))
```

Comment:
How about we append "shared-informers" to the user-agent value here?

Reply:
I thought just identifying the controller manager as a whole would be enough, that is distinct from the other user agents.  I had not compared with upstream.